app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ OPTIONAL ] I think it would be great if you made this method private after using Rails Active callbacks because It is considered good practice to declare callback methods as private. If left public, they can be called from outside of the model and violate the principle of object encapsulation.😊🔥✅✨💯💯🙏🏻You can check this [article](https://guides.rubyonrails.org/active_record_callbacks.html) for more info